seatunnel-connectors-v2/connector-iceberg/pom.xml

```diff
@@ -94,6 +94,7 @@
             <artifactId>hive-exec</artifactId>
             <version>${hive.version}</version>
             <classifier>core</classifier>
+            <scope>provided</scope>
```

Comment:
Add a note about this into iceberg connector docs?

Reply:
ok